lib/facter.rb.in

```diff
@@ -30,8 +30,10 @@ module Facter
       facter_dir = ENV['FACTERDIR'] || File.join(File.expand_path("#{File.dirname(__FILE__)}"), '${LIBFACTER_INSTALL_RELATIVE}')
       # This is a cmake pre-processor check. On *nix it will end up '' == '1'
       # On windows, where we want the changes it will be '1' == '1'
+      # Facter requires the extra inclusion of puppet/bin as the libfacter.so
+      # Lib requires libraries and executables from that directory
       if '${WIN32}' == '1'
-        ENV['PATH'] = "#{File.join(facter_dir, 'bin')}#{File::PATH_SEPARATOR}#{ENV['PATH']}"
+        ENV['PATH'] = "#{File.join(facter_dir, '../puppet/bin')}#{File::PATH_SEPARATOR}#{File.join(facter_dir, 'bin')}#{File::PATH_SEPARATOR}#{ENV['PATH']}"
```

Comment:
I hate assuming a location... but this is probably fine, and shouldn't break if needing a separate directory goes away. As a precaution, I'd put the `../puppet/bin` location second, so it should be ignored if we consolidate to a single location again.